packages/fungible-token/bridge-fungible-token/proxy/src/proxy.sw

```diff
@@ -0,0 +1,153 @@
+contract;
+
+use std::{
+    call_frames::first_param,
+    constants::ZERO_B256,
+    execution::run_external,
+    hash::Hash,
+    hash::sha256,
+    string::String,
+};
+use standards::{src14::SRC14, src20::SRC20, src5::{AccessError, SRC5, State}};
+use contract_message_receiver::MessageReceiver;
+use interface::{bridge::Bridge, src7::{Metadata, SRC7}};
+
+#[namespace(SRC14)]
+storage {
+    // target is at sha256("storage_SRC14_0")
+    target: ContractId = ContractId::zero(),
+    owner: State = State::Initialized(Identity::Address(Address::zero())),
+}
+
+impl SRC5 for Contract {
```

Comment:
This will need to be changed see: https://github.com/FuelLabs/sway-standards/pull/107

Reply:
Changed